packages/server/package.json

```diff
@@ -115,7 +115,7 @@
     "semver": "7.3.2",
     "send": "0.17.1",
     "shell-env": "3.0.1",
-    "signal-exit": "3.0.3",
+    "signal-exit": "3.0.7",
```

Comment:
We are not directly using this dependency. It seems like this was added to be the "top-level" version reference in node_modules. We might be able to remove this...

Reply:
added in this PR: https://github.com/cypress-io/cypress/pull/17215/files

Reply:
@emilyrohrbough Ok, so I was removing this in [this PR](https://github.com/cypress-io/cypress/pull/28975) and the tests did pass, but then I saw it used here after fixing my vscode search: https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/util/exit.js

But it looks like this is actually only referenced in this test file, so should this be a devDep? https://github.com/cypress-io/cypress/blob/develop/packages/server/test/unit/util/file_spec.js#L31

But if it's used in our tests, how did that test ever pass? I've gotten so confused on this one.

Reply:
Ahh it's here: https://github.com/cypress-io/cypress/blob/3c0c864fa9b5d09b5e71c9febaf25ee55bea4374/packages/server/lib/util/file.js#L36

Reply:
Looks like I fell victim to an outdated search from my branch being outdated. Looks like my .gitignore still incldued server/lib* files